charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -117,6 +117,15 @@ data:
   {{- end }}
 {{- end }}
 
+{{- if .Values.sessionProperties }}
+  session-property-config.properties: |
+    session-property-config.configuration-manager=file
```

Comment:
I was just reviewing this again, as I was just about to merge it, and I realized this is limiting if someone would like to use a custom session properties manager. Can you model this in the same way how resource groups are defined? They also allow specifying a JSON configuration, but there's a fallback to a list of properties.

Reply:
sure thing on it

Reply:
@nineinchnick I updated as per your suggestions